File: scraper.js
const urls = [
  "https://www.ivermectinkart.com/product-page/sebifin-cream-terbinafine",
  "https://www.ivermectinkart.com/product-page/nailrox-nail-lacquer-ciclopirox",
  "https://www.ivermectinkart.com/product-page/ketocip-200-mg",
  "https://www.ivermectinkart.com/product-page/ebernet-cream",
  "https://www.ivermectinkart.com/product-page/clocip-b-cream-clotrimazole-beclometasone",
  "https://www.ivermectinkart.com/product-page/canesten-vaginal-cream-clotrimazole",
  "https://www.ivermectinkart.com/product-page/canesten-v6-tablet-vt",
  "https://www.ivermectinkart.com/product-page/fluka-150-mg",
  "https://www.ivermectinkart.com/product-page/forcan-50-mg",
  "https://www.ivermectinkart.com/product-page/forcan-200-mg",
  "https://www.ivermectinkart.com/product-page/forcan-150-mg"
];

async function run() {
  for (let url of urls) {
    try {
      const res = await fetch(url);
      const html = await res.text();
      const nameMatch = html.match(/<title>(.*?)<\/title>/);
      const priceMatchPattern1 = html.match(/"price":\s*([0-9.]+)/);
      const priceMatchPattern2 = html.match(/"price":\s*"([0-9.]+)"/);
      const priceMatchPattern3 = html.match(/property="product:price:amount" content="([0-9.]+)"/);
      const priceMatchPattern4 = html.match(/itemprop="price" content="([0-9.]+)"/);
      let price = priceMatchPattern4 ? priceMatchPattern4[1] : (priceMatchPattern3 ? priceMatchPattern3[1] : (priceMatchPattern2 ? priceMatchPattern2[1] : (priceMatchPattern1 ? priceMatchPattern1[1] : 'N/A')));
      const descMatch = html.match(/<meta property="og:description" content="(.*?)"/);
      const imgMatch = html.match(/<meta property="og:image" content="(.*?)"/);
      
      console.log(`URL: ${url}`);
      console.log(`Title: ${nameMatch ? nameMatch[1] : 'N/A'}`);
      console.log(`Price: ${price}`);
      console.log(`Desc: ${descMatch ? descMatch[1] : 'N/A'}`);
      console.log(`Img: ${imgMatch ? imgMatch[1] : 'N/A'}`);
      console.log('---');
    } catch(e) {
      console.log(`Failed for ${url}: ${e.message}`);
    }
  }
}
run();

File: scraper.py
import urllib.request
import re

urls = [
  "https://www.ivermectinkart.com/product-page/sebifin-cream-terbinafine",
  "https://www.ivermectinkart.com/product-page/nailrox-nail-lacquer-ciclopirox",
  "https://www.ivermectinkart.com/product-page/ketocip-200-mg",
  "https://www.ivermectinkart.com/product-page/ebernet-cream",
  "https://www.ivermectinkart.com/product-page/clocip-b-cream-clotrimazole-beclometasone",
  "https://www.ivermectinkart.com/product-page/canesten-vaginal-cream-clotrimazole",
  "https://www.ivermectinkart.com/product-page/canesten-v6-tablet-vt",
  "https://www.ivermectinkart.com/product-page/fluka-150-mg",
  "https://www.ivermectinkart.com/product-page/forcan-50-mg",
  "https://www.ivermectinkart.com/product-page/forcan-200-mg",
  "https://www.ivermectinkart.com/product-page/forcan-150-mg"
]

for url in urls:
    try:
        req = urllib.request.Request(url, headers={'User-Agent': 'Mozilla/5.0'})
        html = urllib.request.urlopen(req, timeout=10).read().decode('utf-8')
        title = re.search(r'<meta property="og:title" content="(.*?)"', html)
        price = re.search(r'"price":\s*"?([0-9.]+)"?', html)
        if not price:
            price = re.search(r'property="product:price:amount" content="([0-9.]+)"', html)
        img = re.search(r'<meta property="og:image" content="(.*?)"', html)
        desc = re.search(r'<meta property="og:description" content="(.*?)"', html)
        
        print(f"URL: {url}")
        print(f"Title: {title.group(1) if title else 'N/A'}")
        print(f"Price: {price.group(1) if price else 'N/A'}")
        print(f"Img: {img.group(1) if img else 'N/A'}")
        print(f"Desc: {desc.group(1) if desc else 'N/A'}")
        print("---")
    except Exception as e:
        print(f"Failed {url}: {e}")
